mappings/com/mojang/blaze3d/platform/GlStateManager.mapping

```diff
@@ -90,12 +99,34 @@ CLASS com/mojang/blaze3d/platform/GlStateManager
 		ARG 0 target
 		ARG 1 data
 		ARG 2 usage
+	METHOD _glCopyTexSubImage2D (IIIIIIII)V
+		ARG 0 target
+		ARG 1 level
+		ARG 2 xOffset
+		ARG 3 yOffset
+		ARG 4 x
+		ARG 5 y
+		ARG 6 width
+		ARG 7 height
 	METHOD _glDeleteBuffers (I)V
 		ARG 0 buffer
 	METHOD _glDeleteFramebuffers (I)V
 		ARG 0 framebuffer
+	METHOD _glDeleteRenderbuffers (I)V
+		ARG 0 renderbuffers
```

Comment:
Is this inconsistent with the singular arguments of `_glDeleteBuffers` and `_glDeleteFramebuffers`?

Reply:
```suggestion
		ARG 0 renderbuffer
```
I believe so.